### PR TITLE
Subtomo phantom with shifts

### PR DESCRIPTION
**Pull request by pconesa**
This is a mix pull request

On one side it adds shifts to subtomo phantom create a protocol with some tweaks.

On the other side, it composes a name for the resized tomogram similar to the input one.

### CODE REVIEW

**Comment by pconesa** [comment]
After the merge only phantom changes survived!

File: xmipptomo/protocols/protocol_phantom_subtomo.py [hunk 1]
```diff
@@ -49,28 +49,39 @@ def _defineParams(self, form):
...
+        # Shifts
+        form.addSection(label='Shifts')
+        form.addParam('xmin', IntParam, label='Min shift in x', default=0)
+        form.addParam('xmax', IntParam, label='Max shift in x', default=5)
+        form.addParam('ymin', IntParam, label='Min shift in y', default=0)
+        form.addParam('ymax', IntParam, label='Max shift in y', default=5)
+        form.addParam('zmin', IntParam, label='Min shift in z', default=0)
+        form.addParam('zmax', IntParam, label='Max shift in z', default=5)
+
+        form.addSection(label='Others')
```
Comment (EstrellaFG) [inline xmipptomo/protocols/protocol_phantom_subtomo.py:84]: Maybe we can call this section "coordinates" or just "coords". This is just a suggestion, as you prefer.

File: xmipptomo/protocols/protocol_phantom_subtomo.py [hunk 2]
```diff
@@ -133,6 +144,7 @@ def createPhantomsStep(self):
         acq = TomoAcquisition()
         acq.setAngleMax(mwangle)
         acq.setAngleMin(mwangle*-1)
+        subtomobase.setObjComment("Base volume")
```
Comment (EstrellaFG) [inline xmipptomo/protocols/protocol_phantom_subtomo.py:147]: This comments seem nice and really useful!

**Review by EstrellaFG — COMMENTED** [review]
Maybe it is a good idea to extend the test of this protocol with the new functionality. The test is here: https://github.com/I2PC/scipion-em-xmipptomo/blob/devel/xmipptomo/tests/test_protocols_xmipptomo.py#L243
For the rest, I like the new changes

**Comment by pconesa** [comment]
Thanks Estrella for the quick response. Since default values for shifts are active, the test is already using the shifts. Testing the output would be nice, but since the shifts and angles are random it is impossible to predict. I'll rename the section Other with "Coordinates"

**Review by EstrellaFG — APPROVED** [review]

**Pull request merged**